crm_timesheet/models/crm_phonecall.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# License AGPL-3 - See LICENSE file on root folder for details
+##############################################################################
+
+from openerp import models, fields, api
+from datetime import datetime
+
+import logging
+from pprint import pformat
+_logger = logging.getLogger(__name__)
+
+
+class CrmPhonecall(models.Model):
+    _inherit = 'crm.phonecall'
+
+    analytic_account = fields.Many2one(
+        comodel_name='account.analytic.account')
+    timesheets = fields.One2many(comodel_name='hr.analytic.timesheet',
+                                 inverse_name='phonecall')
+
+    def _get_timesheet_vals(self, vals):
```

Comment:
This method should be called by convention `_prepare_timesheet_vals`. Document also what is expected to come in the param vals